fairscale/optim/oss.py

```diff
@@ -285,17 +285,17 @@ def consolidate_state_dict(self, recipient_rank: int = 0) -> None:
                     if should_send_state
                     else torch.tensor([0], dtype=torch.uint8, device=dist_device)
                 )
-                broadcast_object(
-                    state_to_share, src_rank=self.global_rank, group=self.group, dist_device=dist_device,
+                obj_list = [state_to_share]
```

Comment:
HI @blefaudeux, I am clean up the call of broadcast_object since we now have retired the older pytorch version that doesn't support it. However, it is causing some oss test failure. Can you help me check what might be wrong with the cleanup change?

Here is the CI failure:

I ran the failed tests locally and it worked fine. :-(

Reply:
Oh it looks good from a quick check, could that be a backend issue (gloo/nccl) ?

Reply:
I'm guessing that this should work for everything, not just tensors, right ?